src/rt_propagation_types.F

```diff
@@ -255,6 +257,26 @@ SUBROUTINE rt_prop_create(rtp, mos, mpools, dft_control, template, linear_scalin
 
    END SUBROUTINE rt_prop_create
 
+! **************************************************************************************************
+!> \brief Initialize SinvH_imag for rtp
+!> \param rtp ...
+!> \param nspins ...
+! **************************************************************************************************
+   SUBROUTINE rtp_create_SinvH_imag(rtp, nspins)
+      TYPE(rt_prop_type), POINTER                        :: rtp
```

Comment:
I think it is better to remove the `POINTER` attribute which is not required here. Instead, you may add INTENT(INOUT) for better documentation.

Reply:
Thank for the help :)